README.md

```diff
@@ -169,6 +169,17 @@ Using the above configuration file, we now can run a simplified version of the m
 skipper make tests
 ```
 
+### Published ports
+For `shell`, `run` & `make` commands:  
```

Comment:
Consider adding this option only works when build-container-net != host

Reply:
docker run command ignored this option and log a WARNING so have nothing to do....